src/rect.rs

```diff
@@ -500,11 +501,8 @@ impl<T: en::Num> Rect<T> {
         self.outset(T::zero(), T::zero(), T::zero(), left)
     }
 
-    pub fn transform(self, transform: Transform<T>) -> Self {
-        Self::from_iter(
-            self.clockwise_points()
-                .map(|point| point.transform(transform)),
-        )
+    pub fn transform(self, transform: Transform<T>) -> Quad<T> {
```

Comment:
My only concern would be this breaking change here, which I think is the only breaking change in the PR? I don't really see it as a problem, but maybe we also want something like a `transform_aabb` function to make it as painless as possible?

Reply:
There's one more breaking change (the signature of `Vector::transform`), both of which I listed out in CHANGELOG.md. That already had a pre-existing breaking change listed out in the "unreleased" section, which I took as a sign that I could break a couple minor things like this. Based on reading the rest of the changelog, it seems like we haven't been (and IMO we shouldn't be) too afraid of making breaking changes in gee, especially to the point of adding API cruft to deal with them.